src/main/java/sirius/tagliatelle/emitter/BlockEmitter.java

```diff
@@ -42,10 +42,20 @@ public BlockEmitter(Position startOfBlock, String name, Emitter alternative) {
 
     @Override
     protected void emitToContext(LocalRenderContext context) throws Exception {
+        if (context.blockExists(name)) {
+            emitDebugMessage(context, "start rendering block '" + name + "'");
+        } else {
+            emitDebugMessage(context, "requested block '" + name + "' does not exist");
+        }
+
         if (!context.emitBlock(name)) {
             if (alternative != null) {
+                emitDebugMessage(context, "start rendering alternative contents to block '" + name + "'");
```

Comment:
this is local to the template - therefore this might be too verbose(?)

Reply:
I believe it's useful to mark which contents were alternatively rendered when a block did not exist, as in the example below:

```
<i:render name="alt-header">
    <h1>Alternative Header</h1>
</i:render>